scst-sign/release-notes.md

```diff
@@ -0,0 +1,37 @@
+# Supply Chain Security Tools - Sign Release Notes
+
+This topic contains release notes for Supply Chain Security Tools - Sign.
+
+## v1.0.0-beta.2
+
+**Release Date**: November 29, 2021
+
+### New features
+* Added configuration for ResourceQuotas. See `quota.pod_number`
+* Number of replicas can be configured via `replicas` value
+
+### Known issues
+
+* A grype scan has reported the following false positives:
+  * CVE-2015-5237
```

Comment:
Could we add a small blurb on why it's a false positive?

CVE-2015-5237 - This is a CVE on the C implementation of protobuff. We use the Golang version.
CVE-2017-7297 - This is a CVE on rancher server which is not a dependency we include.

Reply:
Great, I will change it.